.github/workflows/mbsolr-tests.yml

```diff
@@ -0,0 +1,28 @@
+name: MusicBrainz Solr Packaging Tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Set up Eclipse Temurin 17
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
```

Comment:
We should also probably supply the paths to all pom.xml files to this, using `cache-dependency-path`. https://github.com/actions/setup-java?tab=readme-ov-file#caching-packages-dependencies

Reply:
```suggestion
          java-version: '17'
          cache: 'maven'
```

Reply:
Because the `pom.xml` files are not located at the top, we'll need to specify the patht manually.

```suggestion
          java-version: '17'
          cache: 'maven'
          cache-dependency-path: |
            mmd-schema/brainz-mmd2-jaxb/pom.xml
            mb-solr/pom.xml
```